README.md

```diff
@@ -1,35 +1,18 @@
-# Enterprise Systems API Sample Application
+# ES-API Sample Apps
 
 Copyright © Bentley Systems, Incorporated. All rights reserved.
 
-A sample application that demonstrates how to create, query and delete a Project using the Enterprise Systems Project API. It also demonstrates Enterprise Systems Work Area Connection API, billing country endpoint and data center endpoint usage.
+A repository for sample applications which utilise aspects of Bentley's ES-API. This README details processes which are the same among the subprojects.
+* [ES-API Projects](/EsApiProjectsSampleApp/) - Demonstrates how to create, query, and delete a Project using the Enterprise Systems Project API.
+* [ES-API WorkArea](/EsApiProjectsSampleApp/) - Demonstrates the Work Area Connection API, which deals with billing country endpoints and data center endpoint usage.
```

Comment:
Billing Country and Data Center is from Projects API. @Woyken can you provide a description for Work Area API?

Reply:
ProjectWise Web Connections Api for mapping Work Area Connections from Project Wise Design Integration to iTwin Projects